<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
